experiments/calibration/run_calibration.jl

```diff
@@ -22,11 +22,26 @@ const CALIBRATE_CONFIG = CalibrateConfig(;
     rng_seed = 42,
 )
 
+# true solution is at 0.96
+priors =
+    [EKP.constrained_gaussian("emissivity_bare_soil", 0.82, 0.12, 0.0, 2.0)]
+prior = EKP.combine_distributions(priors)
+ekp_process = EKP.Unscented(prior)
+ensemble_size = ekp_process.N_ens
 
 if abspath(PROGRAM_FILE) == @__FILE__
-    # Note: Using this script on Derecho requires changes to addprocs to use
-    # the PBSManager
-    addprocs(ClimaCalibrate.SlurmManager())
+    # addprocs is slightly different on derecho or central. Could add more.
+    if contains(gethostname(), "derecho") # Derecho
```

Comment:
```suggestion
    if contains(gethostname(), "derecho") # Derecho
```

I think we need a more elegant solution to this problem in ClimaCalibrate, but this is slightly better for now:
```suggestion
    if get_backend() == ClimaCalibrate.DerechoBackend
```